CharlieBackend.Business/Services/FileServices/ExportFileServices/ExportFactoryService/ExportFactoryService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using CharlieBackend.Business.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CharlieBackend.Business.Services.FileServices.ExportFileServices
+{
+    public class ExportFactoryService : IExportFactoryService
+    {
+        public IExportService GetExportService(ExportFileExtension fileExtension)
```

Comment:
Please create a dictionary `_exportServiceTypes` where the key is file extension and value is typeof([Export Service]). Inject IServiceProvider. Then in the method just return `var serviceType = _exportServiceTypes[fileExtension]; return _serviceProvider.GetRequiredService(serviceType);`.